server/controllers/searchController.js

```diff
@@ -0,0 +1,85 @@
+const Recipe = require("../models/recipeModel");
+const ChefProfile = require("../models/chefProfileModel");
+
+/**
+ * @description Filter through and retrieve recipes or chefs based on filters
+ * @route GET /api/search
+ * @access Public
+ * @params Please refer to the README.md in /server for information on how to use it
+ */
+const getFiltered = async (req, res) => {
+  try {
+    const mongoQuery = {};
+    let filteredOutput = {};
+
+    const cuisinesTagsParam = req.query.cuisines
+      ? JSON.parse(req.query.cuisines)
+      : [];
```

Comment:
Rather than stringifying an array and parsing it, it would match conventions better to structure the query like this `?cuisines=japanese,indian` and then running `req.query.cuisines.split(",")`